Dockerfile

```diff
@@ -73,4 +73,4 @@ COPY . .
 ARG DALI_BUILD_DIR=build-docker-release
 WORKDIR /opt/dali/${DALI_BUILD_DIR}
 
-RUN /opt/dali/docker/build_helper.sh
+RUN bash /opt/dali/docker/build_helper.sh
```

Comment:
You probably can `bash -ex` here as well.

You can also use
`SHELL ["/bin/bash", "-exc"]`
to switch to bash for all run commands.

Reply:
Having this set inside build_helper.sh is better as build.sh can utilize this as well (and any other place which calls/ will call this script).
I don't know if we need to make it global `SHELL ["/bin/bash", "-exc"]`, it is better to avoid any unexpected side effects.

Reply:
Sure, just wanted to discuss other options.